proto/cosmos/group/v1beta1/query.proto

```diff
@@ -32,24 +32,24 @@ service Query {
     option (google.api.http).get = "/cosmos/group/v1beta1/groups_by_admin/{admin}";
   };
 
-  // GroupAccountsByGroup queries group accounts by group id.
-  rpc GroupAccountsByGroup(QueryGroupAccountsByGroupRequest) returns (QueryGroupAccountsByGroupResponse) {
-    option (google.api.http).get = "/cosmos/group/v1beta1/group_accounts_by_group/{group_id}";
+  // GroupPoliciesByGroup queries group policies by group id.
+  rpc GroupPoliciesByGroup(QueryGroupPoliciesByGroupRequest) returns (QueryGroupPoliciesByGroupResponse) {
```

Comment:
Did we decide to remove id suffix? I feel comfortable with id suffix
```suggestion
  rpc GroupPoliciesByGroupId(QueryGroupPoliciesByGroupIdRequest) returns (QueryGroupPoliciesByGroupIdResponse) {
```

Reply:
in this case we would need to do the same for things like `VoteByProposalVoter` which would become `VoteByProposalIdVoterAddress` which seems a bit too long